commands/core/config.drush.inc

```diff
@@ -104,6 +106,7 @@ function config_drush_command() {
         'example-value' => 'branchname',
       ),
       'destination' => 'An arbitrary directory that should receive the exported files. An alternative to label argument',
+      'ignored-modules' => 'A list of modules to ignore during export (e.g. to avoid listing dev-only modules in exported configuration).',
```

Comment:
FYI, _drush_add_commandfiles() already honors an option of the same name. We might have some unintended consequences if we reuse that here. If we do reuse it, I think this should be declared with 'global option' instead of 'option'